src/common/opportunity/prompt.ts

```diff
@@ -6,19 +6,20 @@ import {
 import type { OpportunityJob } from '../../entity/opportunities/OpportunityJob';
 import { textFromEnumValue } from '../protobuf';
 
-export const createOpportunityPrompt = ({
+export const createOpportunityPrompt = async ({
   opportunity,
 }: {
   opportunity: OpportunityJob;
 }) => {
+  const locations = await opportunity.locations;
+  const firstLocation = locations?.[0];
```

Comment:
where are we planning to do with other locations? (i know that it used to be this way even before this PR)